blog/2020-07-31-temporal-v0.28.0-changelog.md

```diff
@@ -0,0 +1,111 @@
+        - Renamed:`DecisionTaskStartToCloseTimeout` → `WorkflowTaskTimeout`. Defaults to 10 seconds.
+- **Workflow retry refactors** 🔁 :
+    - `ExpirationInterval` has been removed from `RetryPolicy`. Instead, `WorkflowExecutionTimeout` and `ScheduleToClose` are used to stop retries for Workflows and Activities respectively.
+- **Activity retry refactors** 🔁 :
+    - Temporal includes default `RetryOptions` for Activities.
+    - To disable Activity retries, configure Activities with `RetryOptions` that set `maximumAttempts` to 1.
+- **tctl (CLI) updates** 🧰 :
+    - New admin commands:
+        - List namespaces ([#463](https://github.com/temporalio/temporal/pull/463)): `"namespace list"`
+        - Describe Shard ([#370](https://github.com/temporalio/temporal/pull/370)): `"shard describe --shard_id ..."` / `"d -sid"`
+        - List Tasks ([#429](https://github.com/temporalio/temporal/pull/429)/[#433](https://github.com/temporalio/temporal/pull/433)): `shard list_tasks`
+        - Describe Task ([#408](https://github.com/temporalio/temporal/pull/408), [#416](https://github.com/temporalio/temporal/pull/416), [#412](https://github.com/temporalio/temporal/pull/412)): `"shard describe_task --task_type [timer, replication, transfer] ..."` / `"dt"`
+        - List Cluster memberships([#423](https://github.com/temporalio/temporal/pull/423/files)): `"membership list_db"`
+        - List Ringpop members ([#426](https://github.com/temporalio/temporal/pull/426)): `"membership list_gossip"`
+        - List Replication DLQ Tasks ([#456](https://github.com/temporalio/temporal/pull/456)): `"kafka list_dlq"`
+    - Admin Command improvements:
+        - Added filtering of Tasks from TaskQueue based on Workflow and run Ids  ([#462](https://github.com/temporalio/temporal/pull/462))
+        - Consistency to CLI admin command names ([#427](https://github.com/temporalio/temporal/pull/427))
+        - Parameter types changed from enum to string ([#445](https://github.com/temporalio/temporal/pull/445), [#447](https://github.com/temporalio/temporal/pull/447))
+        - Table-view support for CLI list commands ([#444](https://github.com/temporalio/temporal/pull/444))
+    - Fixes
+        - Admin cluster command for adding search attributes to be indexed ([#514](https://github.com/temporalio/temporal/pull/514))
+        - Non-string heartbeat details fail to display and only the first heartbeat detail shows up in CLI ([#525](https://github.com/temporalio/temporal/pull/525))
+        - Printing Query CLI command results ([#542](https://github.com/temporalio/temporal/pull/542))
+        - CLI reuse policy flag type ([#473](https://github.com/temporalio/temporal/pull/473))
+    - House cleaning:
+        - Remove executable permission from JSON files ([#483](https://github.com/temporalio/temporal/pull/483))
+- **Error chaining** ⛓️  :  Errors can now be chained together across SDKs in different languages and structured as Protobuf messages.
```

Comment:
Add:
- Errors are serialized using special proto type `Failure` and can be transmitted between different languages.

Maybe this is also for protobuf section on top.